### PR TITLE
remove env vars from dynamo for new releases, only use encrypted s3

### PR DESCRIPTION
**Pull request by MiguelMoll**

### CODE REVIEW

**Comment by MiguelMoll** [comment]
[Encrypt env in dynamo](https://trello.com/c/euahY6OX/2212-encrypt-env-in-dynamo)

**Comment by nzoschke** [comment]
Patch looks good. This one should have some careful QA. Perhaps something like:

* On old version, create app and create release (R1) with `convox env set FOO=bar`
* Deploy app and verify `convox exec env`
* Update rack
* Verify `convox env`
* Create new release (R2) with `convox env set FOO=baz`
* Deploy app and verify `convox exec env`
* Roll back to R1 and verify `convox exec env`
* Roll forward to R2 and verify `convox exec env`

**Comment by MiguelMoll** [comment]
Checklist plays out as expected. 👍

File: provider/aws/releases.go [hunk 1]
```diff
@@ -45,7 +45,30 @@ func (p *AWSProvider) ReleaseGet(app, id string) (*structs.Release, error) {
...
+	if err != nil {
+		return nil, err
+	}
+
+	if a.Parameters["Key"] != "" {
+		cr := crypt.New(p.Region, p.Access, p.Secret)
+
+		if d, err := cr.Decrypt(a.Parameters["Key"], data); err == nil {
+			data = d
+		}
```
Comment (nzoschke) [inline provider/aws/releases.go]: is this missing an error check?

**Pull request closed without merging**